cmd/dfget/app/root.go

```diff
@@ -165,63 +174,61 @@ func initClientLog() error {
 	flagSet.IntVarP(&cfg.Timeout, "timeout", "e", 0,
-		"download timeout(second)")
+		"Timeout set for file downloading task. If dfget has not finished downloading all pieces of file before --timeout, the dfget will throw an error and exit")
 
 	// md5 & identifier
 	flagSet.StringVarP(&cfg.Md5, "md5", "m", "",
-		"expected file md5")
+		"md5 value input from user for the requested downloading file to enhance security")
 	flagSet.StringVarP(&cfg.Identifier, "identifier", "i", "",
-		"identify download task, it is available merely when md5 param not exist")
-
+		"The usage of identifier is making different downloading tasks generate different downloading task IDs even if they have the same URLs. conflict with --md5.")
 	flagSet.StringVar(&cfg.CallSystem, "callsystem", "",
-		"system name that executes dfget")
+		"The name of dfget caller which is for debugging. Once set, it will be passed to all components around the request to make debugging easy")
 
 	flagSet.StringVarP(&cfg.Pattern, "pattern", "p", "p2p",
-		"download pattern, must be 'p2p' or 'cdn' or 'source'"+
-			"\ncdn/source pattern not support 'totallimit' flag")
+		"download pattern, must be p2p/cdn/source, cdn and source do not support flag --totallimit")
 
 	flagSet.StringVarP(&filter, "filter", "f", "",
-		"filter some query params of url, use char '&' to separate different params"+
+		"filter some query params of URL, use char '&' to separate different params"+
 			"\neg: -f 'key&sign' will filter 'key' and 'sign' query param"+
-			"\nin this way, different urls correspond one same download task that can use p2p mode")
+			"\nin this way, different but actually the same URLs can reuse the same downloading task")
 
 	flagSet.StringSliceVar(&cfg.Header, "header", nil,
 		"http header, eg: --header='Accept: *' --header='Host: abc'")
 
 	flagSet.StringSliceVarP(&cfg.Node, "node", "n", nil,
-		"specify supnernodes")
+		"specify the addresses(IP:port) of supnernodes")
 
 	flagSet.BoolVar(&cfg.Notbs, "notbs", false,
-		"not back source when p2p fail")
+		"disable back source downloading for requested file when p2p fails to download it")
 	flagSet.BoolVar(&cfg.DFDaemon, "dfdaemon", false,
-		"caller is from dfdaemon")
+		"identify whether the request is from dfdaemon")
 	// others
 	flagSet.BoolVarP(&cfg.ShowBar, "showbar", "b", false,
-		"show progress bar, it's conflict with '--console'")
+		"show progress bar, it is conflict with '--console'")
 	flagSet.BoolVar(&cfg.Console, "console", false,
 		"show log on console, it's conflict with '--showbar'")
 	flagSet.BoolVar(&cfg.Verbose, "verbose", false,
 		"be verbose")
 
 	// pass to server
 	flagSet.DurationVar(&cfg.RV.DataExpireTime, "expiretime", config.DataExpireTime,
-		"server will delete cached files if these files doesn't be modification within this duration")
+		"caching duration for which cached file keeps no accessed by any process, after this period cache file will be deleted")
```

Comment:
`expiretime` and `alivetime` root flags are also flags of `dfget server`, they should be modified in the same time.

Reply:
updated.